lint.sh

```diff
@@ -5,7 +5,8 @@ set -e
 FWDIR="$(cd "`dirname $0`"; pwd)"
 cd "$FWDIR"
 
-pycodestyle --max-line-length=100 --exclude mlflow/protos,mlflow/server/js,mlflow/store/db_migrations,mlflow/temporary_db_migrations_for_pre_1_users -- mlflow tests
+black --check --line-length=100 --exclude=mlflow/protos .
+pycodestyle --max-line-length=100 --ignore=E203,W503 --exclude mlflow/protos,mlflow/server/js,mlflow/store/db_migrations,mlflow/temporary_db_migrations_for_pre_1_users -- mlflow tests
```

Comment:
If we're going to check lint and encourage autoformatting via `black`, maybe we can remove the `pycodestyle` check? Let's do it in a follow-up PR so that we can merge this one first :)